src/test/java/com/CourseScheduleBuilder/courseApplicationTests.java

```diff
@@ -8,7 +9,7 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class courseApplicationTests {
-
+	@Ignore
```

Comment:
I think we can delete this whole test class. Will make whole new tests probably for next weekend, they're easy to do